examples/cython/cython_examples/__init__.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
```

Comment:
In all of the new files, can we use all three

```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

Reply:
Makes sense. I held off for the *.pyx files, but included an additional note in the documentation about Python 2 vs. 3 + where folks can get more information.